src/main/java/seedu/duke/Dish.java

```diff
@@ -0,0 +1,29 @@
+package seedu.duke;
```

Comment:
I think better to move this into its own folder. Probably can name the folder `entities` or something.

Reply:
If we need to split to different types of entities (e.g. split Staff classes and Dish classes), then within the `entities` folder we can just add more folders haha.